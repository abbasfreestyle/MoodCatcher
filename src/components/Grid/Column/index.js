import React from 'react';
import { View } from 'react-native';

const Column = props => <View flexDirection="column" {...props} />;

export default Column;
