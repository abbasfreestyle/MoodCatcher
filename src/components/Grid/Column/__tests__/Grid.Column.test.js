import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Column from '..';

const wrapper = shallow(<Column />);

describe('Grid > Column.', () => {
  describe('Snapshots.', () => {
    test('should match', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
