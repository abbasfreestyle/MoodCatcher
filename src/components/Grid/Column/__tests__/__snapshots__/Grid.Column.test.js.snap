// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid > Column. Snapshots. should match 1`] = `
<View
  flexDirection="column"
/>
`;
